recipes/recipes_emscripten/harfbuzz/build.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+export LDFLAGS="$(LDFLAGS) -sUSE_PTHREADS=0"
+
```

Comment:
Same here.

https://github.com/harfbuzz/harfbuzz/blob/main/meson.build#L368

We need to patch the meson file to disable threads.

Reply:
Done